.goreleaser.yml

```diff
@@ -38,9 +38,14 @@ builds:
 
 archives:
 -
-  replacements:
-    darwin: macOS
-    386: i386
+  rlcp: true
+  name_template: >-
+    {{ .ProjectName }}_
+    {{- if eq .Os "darwin" }}macOS
+    {{- else }}{{ .Os }}{{ end }}_
+    {{- if eq .Arch "386" }}i386
+    {{- else }}{{ .Arch }}{{ end }}
+    {{- if .Arm }}v{{ .Arm }}{{ end }}
```

Comment:
Will this `Arm` line change the existing naming scheme?

Reply:
Good callout, I meant to add a comment that I need to test this. The goreleaser docs said that these templates would be equivalent but I just did some testing locally with `goreleaser release --snapshot` and I'm getting different artifacts before and after this change. I'll need to take a closer look at this

Reply:
I force pushed a change and I think I've got it!

I pulled the defaults from https://goreleaser.com/customization/nfpm/?h=nfpms and https://goreleaser.com/customization/archive/?h=archives to make the necessary modifications.

I ran `rm -rf dist && goreleaser release --snapshot; (find dist | sort)` to get the generated list of artifacts and they look identical apart from the version hashes 👍 

<img width="1624" alt="Screenshot 2023-02-16 at 5 19 50 PM" src="https://user-images.githubusercontent.com/2954553/219500051-c22f95af-ae2a-4434-9ae6-feb453c2e497.png">

Reply:
Nice verification! I much preferred goreleaser's old method for defining this, but so is life...

Reply:
Agreed 😞